src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: bold;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+:root {
+  --big-text-padding-left: 62px;
+  --small-text-padding-left: 33px;
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
```

Comment:
don't add `font-family` and `font-size` only for placeholder, add it for whole base class, move it to `.input`
```suggestion
```
![image](https://user-images.githubusercontent.com/37161315/193046619-8ed3251c-d8f9-42a5-aec6-ce55da48b67e.png)